src/safeds/ml/classical/classification/_baseline_classifier.py

```diff
@@ -2,6 +2,8 @@
 from concurrent.futures import ALL_COMPLETED, wait
 from typing import Self
 
+from joblib._multiprocessing_helpers import mp
```

Comment:
```suggestion
import multiprocessing as mp
```

IDEs can handle this direct import better.